rest/admin_api.go

```diff
@@ -30,6 +30,10 @@ const kDefaultDBOnlineDelay = 0
 
 const paramDisableOIDCValidation = "disable_oidc_validation"
 
+// GetUsers  - GET /{db}/_user/
+const paramNameOnly = "name_only"
+const paramLimit = "limit"
```

Comment:
Can these parameters be included in the API docs?

You can add define them in the `docs/api/components/parameters.yaml` file

```yaml
usersNameOnly:
  name: name_only
  in: query
  required: false
  schema:
    type: boolean
    default: true
  description: Whether to return user names only, or more detailed information for each user.
usersLimit:
  name: limit
  in: query
  required: false
  schema:
    type: integer
  description: How many results to return.
```

and then reference them from the `docs/api/paths/admin/{db}~_user~.yaml` file on the GET method:

```yaml
get:
  parameters:
    - $ref: ../../components/parameters.yaml#/usersNameOnly
    - $ref: ../../components/parameters.yaml#/usersLimit
```

Reply:
Will do - thanks for typing that up.